packages/payment-integration-api/src/payment/payment.ts

```diff
@@ -22,6 +23,7 @@ export type PaymentInstrument =
           | AdyenV2Instrument
           | AppleInstrument
           | BlueSnapDirectCreditCardInstrument
+          | BlueSnapDirectEcpPayload
```

Comment:
In which way is this type different from `BlueSnapDirectEcpInstrument` ?

Reply:
```js
// This is how we receive the data from the client e.g. checkout-js payment form
interface BlueSnapDirectEcpInstrument {
    accountNumber: string;
    accountType: BlueSnapDirectEcpAccountType;
    shopperPermission: boolean;
    routingNumber: string;
}

// This is how we need to send the data to the backend so that it understands it.
interface BlueSnapDirectEcpPayload {
     // Wrapped on an ecp property
    ecp: {
        // Each property snake_case
        account_number: string;
        account_type: BlueSnapDirectEcpAccountType;
        shopper_permission: boolean;
        routing_number: string;
    };
}
```